API/Services/Implementations/PlayerRatingsService.cs

```diff
@@ -45,7 +45,4 @@ IMapper mapper
             Adjustments = mapper.Map<ICollection<RatingAdjustmentDTO>>(currentStats.Adjustments.OrderBy(a => a.Timestamp))
         };
     }
-
-    public async Task<IDictionary<int, int>> GetHistogramAsync(Ruleset ruleset) =>
```

Comment:
Forgive me if I'm missing something, but can you remind me why this was removed?

Reply:
Ah right, yep